.circleci/config.yml

```diff
@@ -3,24 +3,37 @@ workflows:
   version: 2
   flow:
     jobs:
-      - deploy:
+      - release-libraries:
           context: babylon
           filters:
             branches:
               only:
                 - 7.x
+      - deploy-service-dev:
+          context: babylon
+          filters:
+            branches:
+              only:
+                - 7.x
+      - deploy-service-staging:
+          context: babylon
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^.*/
```

Comment:
what happens when we merge B2I code into our forked project? we might receive some git tags from them :D

Reply:
maybe add a `babylon-` prefix to have a easy to reach PATTERN

Reply:
adding bbl-